modules/internal/OwnedObject.chpl

```diff
@@ -208,6 +208,10 @@ module OwnedObject {
   proc =(ref lhs:_owned, rhs:_nilType) {
     lhs.clear();
   }
+  pragma "no doc"
```

Comment:
can we chpldoc it?

Reply:
We could.  I was following the lead of the assignment overloads (in both files).

Reply:
Oh, I see, owned = is not chpldoc'd. This is fine for now then.